.travis.yml

```diff
@@ -1,8 +1,32 @@
 language: ruby
-cache: bundler
+cache:
+  bundler: true
+  yarn: true
+  directories:
+  - node_modules
+  - vendor/bundle
+install:
+- bundle install --deployment
+- yarn install
 before_script:
-  - "export SECRET_KEY_BASE=`bundle exec rake secret`"
-  - "bundle exec rake db:create RAILS_ENV=test"
+- export SECRET_KEY_BASE=`bundle exec rake secret`
+- bundle exec rake db:create RAILS_ENV=test
+script:
+- bundle exec rake
+- yarn test
+after_success:
+  - chmod +x ./compile-build
+  - "./compile-build"
+deploy:
+  provider: releases
```

Comment:
Refactor into using Gitlab Docker image creation

Reply:
Ok, gitlab config for continuous deployment development will be added as a new user story